scipy/stats/_morestats.py

```diff
@@ -3679,6 +3686,12 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
     WilcoxonResult(statistic=6.0, pvalue=0.4375)
 
     """
+    # replace approx by asymptotic to ensure backwards compatability
```

Comment:
I know we discussed this, but I just remembered that we allow both `mode` and `method` for this argument, so there would be some precedent if we want to just accept "approx" as an alias rather than being disruptive.